src/pages/Login/Signup.components.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import './Signup.styles.scss';
+
```

Comment:
Why did you name Signup.components if Signup is a page? I suggest renaming it to Signup

Reply:
You fixed that.